packages/cli/src/controllers/passwordReset.controller.ts

```diff
@@ -120,7 +120,7 @@ export class PasswordResetController {
 		const resetPasswordToken = this.jwtService.signData(
 			{ sub: id },
 			{
-				expiresIn: '1d',
+				expiresIn: '20m',
```

Comment:
love it.. so simple.. can you double check that the email does not say anything about a day expiry?

Reply:
Will check 🫡

Reply:
Nice catch! It actually said 2 hours. I amended the change to the same commit as this was so tiny change. Also rebased to master